core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -146,375 +148,375 @@ public static SpanExpression span(Expression field, Expression value, String uni
   }
 
   public static FunctionExpression abs(Expression... expressions) {
-    return compile(BuiltinFunctionName.ABS, expressions);
+    return compile(FunctionProperties.None, BuiltinFunctionName.ABS, expressions);
```

Comment:
nit, why not add compile helper method instead of change the all these functions?

Reply:
Accessing FunctionProperties.None results in RuntimeException. I wanted to ensure someone doesn't run into that unintentionally. 

I did have a helper method initially. Do you think it's worthwhile to add it back?